README.md

```diff
@@ -46,6 +46,29 @@ python3 setup.py install --with-parallel --with-gslib --user
 ```
 Make sure [`swig`](https://pypi.org/project/swig) is installed first. Also, the binary file must be located in `PATH` environment variable.
 
+### pylibROM-Jupyter Docker Container
+
+This Docker container provides an environment with Jupyter Notebook for the pylibROM library. Follow the steps below to build the Docker image and run a Jupyter Notebook server.
+
+#### Build the Docker Image
+
+Navigate to the directory containing the Dockerfile:
+ ```
+cd /path/to/folder/pylibROM/docker/jupyter_pylibROM
```

Comment:
The folder containing the Dockerfile could probably get renamed to be a bit shorter: `docker/jupyter_pylibROM` to `docker/jupyter`.